mrp_production_manual_close/models/mrp_production.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+# For copyright and license notices, see __openerp__.py file in root directory
+##############################################################################
+
+from openerp import models, fields, api
+
+
+class MrpProduction(models.Model):
+    _inherit = 'mrp.production'
+
+    production_done = fields.Boolean(string="Production Done")
+
+    @api.multi
+    def test_production_done2(self):
```

Comment:
A ver... El tema es que ahora cuando en una of se supera el numero de
unidades a producir se ejecuta el paso de estado de en proceso a realizado.
Lo unico que hay que hacer es interceptar esa señal. Nada mas. Ni nuevos
botones ni checks ni nada de nada.
Ya lo dije en el claim y lo explique tambien en su momento
El 08/07/2015 15:21, "Daniel" <notifications@github.com> escribió:

> In mrp_production_manual_close/models/mrp_production.py
> <https://github.com/odoomrp/odoomrp-wip/pull/899#discussion_r34145649>:
>
> > @@ -0,0 +1,23 @@
> > +# -*- coding: utf-8 -*-
> > +##############################################################################
> > +# For copyright and license notices, see __openerp__.py file in root directory
> > +##############################################################################
> > +
> > +from openerp import models, fields, api
> > +
> > +
> > +class MrpProduction(models.Model):
> > +    _inherit = 'mrp.production'
> > +
> > +    production_done = fields.Boolean(string="Production Done")
> > +
> > +    @api.multi
> > +    def test_production_done2(self):
>
> Correcto, mi idea de poner el campo production_done, era para tener una
> referencia de cuando la producción se ha cumplido con el objeto de poder
> ocultar/mostrar los botones necesarios sin tener que añadir un nuevo
> estado, no se si hay alguna otra forma más sencilla de hacerlo.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/odoomrp/odoomrp-wip/pull/899/files#r34145649>.
>